dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -709,12 +708,20 @@ internal open class RumViewScope(
         val memoryInfo = lastMemoryInfo
         val refreshRateInfo = lastFrameRateInfo
         val isSlowRendered = resolveRefreshRateInfo(refreshRateInfo) ?: false
+        val currentViewId = rumContext.viewId.orEmpty()
```

Comment:
there is no need to have this line. `rumContext` above is already immutable structure and can be used in lambda directly like on the line with:

```
view = ViewEvent.View(
  id = rumContext.viewId.orEmpty()
```

Reply:
yup was a leftover

Reply:
still there